src/Gui/CommandStructure.cpp

```diff
@@ -71,6 +70,14 @@ void StdCmdPart::activated(int iMsg)
     // TODO We really must set label ourselves? (2015-08-17, Fat-Zer)
     doCommand(Doc,"App.activeDocument().%s.Label = '%s'", PartName.c_str(),
             QObject::tr(PartName.c_str()).toUtf8().data());
+
+    doCommand(Doc,
+    "selected_objects = Gui.Selection.getSelection()\n"
+    "if len(selected_objects) > 1:\n"
+    "    for obj in selected_objects:\n"
+    "        App.activeDocument().%s.addObject(obj)",
+    PartName.c_str());
```

Comment:
You can simplify this if you like
```suggestion
    doCommand(Doc,
        "for obj in Gui.Selection.getSelection()[1:]:\n"
        "    App.activeDocument().%s.addObject(obj)",
        PartName.c_str());
```

Reply:
thanks for your help

Reply:
I disagree:
```python
>>> a=[0,1,2,3]
>>> for b in a:
...     print(b)
...     
0
1
2
3
>>> for b in a[1:]:
...     print(b)
...     
1
2
3
```

so the first object is not moved to the Part.

Reply:
@FlachyJoe yikes! Thanks for pointing it out!
I accidentally read it as the first one was to be skipped, which of cause is _not_ what is happening here!